zipline/assets/asset_writer.py

```diff
@@ -432,6 +445,8 @@ def write(self,
                   A short description of this root symbol.
               exchange : str
                   The exchange where this root symbol is traded.
+        supplementary_mappings : pd.DataFrame, optional
+            Additional mappings from values of abitrary type to assets.
```

Comment:
I'm leaning towards just equities, since we already segregate by asset type in all our existing tables holding this sort of data. Any thoughts?